<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Dialog</class>
 <widget class="QWidget" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VerTilgungsPlan - Josh's RatenKreditRechner</string>
  </property>
  <property name="windowIcon" >
   <iconset>icon.xpm</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="4" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="status" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>619</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Print</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>&amp;Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QTextBrowser" name="textBrowser" >
     <property name="font" >
      <font>
       <family>Courier</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Courier; font-size:12pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Starting today: Tue Sep 26 01:00:51 2006&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">SUMME:  55555.000000 euro&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Zinssatz: 6.750000 &lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Tilgung: 5555.000000 euro&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Zahlung/Jahr: 4 / Jahr&lt;/p>&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Der Zins bei der ersten Rate: 937.490000 &lt;/p>&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Rate     Datum        Tilgung       Zins       Betrag&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  1      26-Dec-2006      4617.510000      937.490000      50937.490000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  2      27-Mar-2007      4695.430000      859.570000      46242.060000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  3      26-Jun-2007      4774.670000      780.330000      41467.390000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  4      26-Sep-2007      4855.240000      699.760000      36612.150000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  5      26-Dec-2007      4937.170000      617.830000      31674.980000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  6      26-Mar-2008      5020.480000      534.520000      26654.500000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  7      25-Jun-2008      5105.210000      449.790000      21549.290000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  8      25-Sep-2008      5191.360000      363.640000      16357.930000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">  9      25-Dec-2008      5278.960000      276.040000      11078.970000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"> 10      26-Mar-2009      5368.040000      186.960000      5710.930000&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"> 11      25-Jun-2009      5458.630000      96.370000      252.300000&lt;/p>&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Sum(Zins)::5806.560000, Sum(Tilgung)::55555.000000&lt;/p>&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Kreditbetrag , Zinssatz</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="betrag" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Kreditbetrag, Zinssatz muss angegeben werden</string>
        </property>
        <property name="autoFillBackground" >
         <bool>false</bool>
        </property>
        <property name="suffix" >
         <string>  â‚¬</string>
        </property>
        <property name="maximum" >
         <number>99999999</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="singleStep" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>20274</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Zinssatz:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="zinsen" >
        <property name="toolTip" >
         <string>Zinssatz muss angegeben werden</string>
        </property>
        <property name="suffix" >
         <string> %</string>
        </property>
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="singleStep" >
         <double>0.25</double>
        </property>
        <property name="value" >
         <double>6.75</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Betrag</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Tilgung oder Restlaufzeit berechnen?</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="radioLaufzeit" >
        <property name="toolTip" >
         <string>Restlaufzeit vorgegeben -> Tilgung pro Rate wird berechnet</string>
        </property>
        <property name="text" >
         <string>RestLaufzeit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="radioTilgung" >
        <property name="toolTip" >
         <string>Tilgung pro Rate vorgegeben -> Restlaufzeit wird berechnet</string>
        </property>
        <property name="text" >
         <string>Zahlung [Euro] :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="tilgung" >
        <property name="suffix" >
         <string>  â‚¬</string>
        </property>
        <property name="prefix" >
         <string/>
        </property>
        <property name="maximum" >
         <number>999999999</number>
        </property>
        <property name="singleStep" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>2571</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="restlaufzeit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="maximum" >
         <number>999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Raten / Jahr</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="r3" >
        <property name="text" >
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="r1" >
        <property name="text" >
         <string>12</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="r2" >
        <property name="text" >
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QRadioButton" name="r4" >
        <property name="text" >
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Startdatum</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QDateEdit" name="dateEdit" >
        <property name="toolTip" >
         <string>Startdatum des Ratenkredits</string>
        </property>
        <property name="date" >
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Tipp: Mausrad verwenden</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>r1</tabstop>
  <tabstop>r2</tabstop>
  <tabstop>r3</tabstop>
  <tabstop>r4</tabstop>
  <tabstop>betrag</tabstop>
  <tabstop>zinsen</tabstop>
  <tabstop>radioTilgung</tabstop>
  <tabstop>radioLaufzeit</tabstop>
  <tabstop>tilgung</tabstop>
  <tabstop>restlaufzeit</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>dateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioTilgung</sender>
   <signal>clicked(bool)</signal>
   <receiver>restlaufzeit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>549</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>595</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLaufzeit</sender>
   <signal>clicked(bool)</signal>
   <receiver>tilgung</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>511</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel" >
     <x>633</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioTilgung</sender>
   <signal>clicked(bool)</signal>
   <receiver>tilgung</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>506</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel" >
     <x>595</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLaufzeit</sender>
   <signal>clicked(bool)</signal>
   <receiver>restlaufzeit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>555</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>598</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
